client/components/spotlight/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { Button } from '@automattic/components';
```

Comment:
@Rolilink Let's please add this component documentation before merging the PR.

Reply:
Thank you! I think the only thing missing is

* add the component in client/devdocs/design/index.jsx so that it's visible in https://wpcalypso.wordpress.com/devdocs/design